src/styles/colors.js

```diff
@@ -0,0 +1,61 @@
+const white = "#FFFFFF";
+
+const primaryText = "#1F1F1F";
+
+const primaryBackground = "#F7F8FC";
+
+const darkGradientBackground =
+  "linear-gradient(93.51deg, #182E7C 8.9%, #030D67 94.31%)";
+
+const lightGradientBackground =
+  "linear-gradient(93.51deg, #1782DC 8.9%, #6AB7F7 94.31%)";
+
+const activeButton = "#3678E3";
```

Comment:
Is it for background or a font color? 🙃 Maybe nor relevant actually.

Reply:
It is just for background only.